marketplace_docs/templates/Fabric/fabfile.py

```diff
@@ -102,6 +102,12 @@ def run_scripts():
         run("chmod +x %s" % rfile)
         run(rfile)
 
+def setup_interactive_login_script():
+    run("mkdir /opt/zulip")
+    put("interactive_script.sh", "/opt/zulip/interactive_script.sh")
+    run("chmod +x /opt/zulip/interactive_script.sh")
+    run("cp /root/.bashrc /etc/skel/.zulip_bashrc")
+    run("echo '/opt/zulip/interactive_script.sh' >> /root/.bashrc")
```

Comment:
I assume that this approach of putting something in `.bashrc` is the recommended approach for an interactive script?  Seems janky, but OK if that's what they recommend.

Reply:
Yup. They have recommended it like that. https://github.com/zulip/marketplace-partners/blob/master/marketplace_docs/build-an-image-fabric.md#running-commands-on-first-login